cime/src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -128,14 +170,19 @@ subroutine prep_glc_init(infodata, lnd_c2_glc)
          esmf_map_flag=esmf_map_flag   , &
          glc_present=glc_present       , &
          lnd_gnam=lnd_gnam             , &
-         glc_gnam=glc_gnam)
+         glc_gnam=glc_gnam             , &
+         ocn_gnam=ocn_gnam)
 
     allocate(mapper_Sl2g)
     allocate(mapper_Fl2g)
+    allocate(mapper_So2g)
+    allocate(mapper_Fo2g)
     allocate(mapper_Fg2l)
 
     smb_renormalize = prep_glc_do_renormalize_smb(infodata)
 
+    g2x_gx => component_get_c2x_cx(glc(1)) !Jer: is this needed here?
```

Comment:
@JeremyFyke , can you elaborate on what the potential issue is here?

Reply:
I am not sure, sorry.  Perhaps nothing from this structure is utilized in following code?

Reply:
It appears to be unused, so I removed it.